ckanext/lacounts/jobs.py

```diff
@@ -0,0 +1,51 @@
+import ast
+import time
+import logging
+from ckan import model
+import ckan.plugins.toolkit as toolkit
+from ckanext.lacounts.harvest import helpers
+log = logging.getLogger(__name__)
+
+
+def update_groups(group_name):
+    # This job re-calculate all harvested dataset groups from scratch
+    # It makes actual update call only on changed packages
+    time.sleep(3)
+
+    # Get groups
+    groups = helpers.list_groups_with_extras()
+
+    # Get extras
+    extras = (model.Session
+        .query(model.PackageExtra)
+        .filter(model.PackageExtra.key == 'harvest_dataset_terms')
+        .filter(model.PackageExtra.value != '[]')
+        .all())
+
+    # Get packages
+    offset = 0
+    limit = 1000
+    packages = []
+    while True:
```

Comment:
I really don't want us iterating over all datasets every time the topics are rebuilt. We can merge this as is, but then we can work on an optimization. I'll write an issue to see what you think